### PR TITLE
Add Calculator for Connection/Well Productivity Index

### PR DESCRIPTION
**Pull request by bska**
Switches between using the logarithmic and unit scaling factor based on whether or not the well has an explicit, positive drainage radius (`WELSPECS` item 7).  Does presently not include the D factor.

Add a set of unit tests to exercise the facility.

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
May I request a review of this PR please?

**Comment by bska** [comment]
jenkins build this please

File: opm/simulators/wells/WellProdIndexCalculator.hpp
Comment (atgeirr) [inline opm/simulators/wells/WellProdIndexCalculator.hpp:31]: Please add some documentation explaining the basic idea of the class, the fact that it contains only static well information etc.

**Review by atgeirr — APPROVED** [review]
Good to merge once a little documentation has been added.

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
> jenkins build this please

Self-merging this as the build check is green and the PR was approved by a maintainer.

**Pull request merged**
